resources/pre-upgrade.sh

```diff
@@ -6,6 +6,7 @@ set -o pipefail
 # WORKDIR is a Dockerfile global variable
 REDMINE_WORK_DIR="${WORKDIR}"
 MIGRATION_TMP_DIR="/var/tmp/redmine/plugins/migration4.4.2.1"
+ERROR_SLEEP_IN_S=300
```

Comment:
I think the idea of this is to prevent the dogu from restarting overy few seconds.
But this should not be an actual usecase as this function is not executed in startup, only when you execute an upgrade.
So in case of failure the upgrade command would sleep for 300 seconds. I dont think this is necessary.

Reply:
You might have a good point there. Hmmm. Then a simple `exit 1` without sleep will do. Good catch!

Reply:
- [x] done